docs/pages/product/workspace/semantic-catalog.mdx

```diff
@@ -0,0 +1,59 @@
+# Semantic Catalog
+
+Semantic Catalog lets users search, understand, and re-use trusted data products,
+both from Cube and external sources such as cloud data warehouses and business intelligence tools.
+
+<SuccessBox>
+
+Semantic Catalog is available in Cube Cloud on
+[Premium and above](https://cube.dev/pricing) tiers.
+[Contact us](https://cube.dev/contact) for details.
+
+</SuccessBox>
+
+<Screenshot src="https://ucarecdn.com/d8e09149-7ed2-4f5c-83f6-406734557efb/catalog.jpg" />
+
+## Getting Started
+
+### Enabling Semantic Catalog
+
+Semantic Catalog is currently in preview. First, ask your account team to enable it for you.
+
+Then, in your Cube deployment, navigate to the "Catalog" sidebar tab.
+If it is not already enabled, there will be a button to get started.
+After you click this button, your Cube data model will sync to the Semantic Catalog.
+In a few moments, you will see Cubes, Views, and Tables appear in the Semantic Catalog.
+
+### Connecting downstream tools
+
+In addition to your Cube data model, Semantic Catalog can be used to understand charts and dashboards in downstream business intelligence and data visualization tools.
+These currently include:
+
+- [Tableau][ref-tableau]
+- [Superset][ref-superset]
+- [Preset][ref-superset]
+- Amazon Quicksight
+
+To connect these, navigate to "Settings" in your Cube deployment sidebar, and then choose "Catalog Services."
+Click the "Add Service" button, and a modal will pop up for you to enter credentials, test the connection, and create your service.
+
+<Screenshot src="https://ucarecdn.com/a060a3cb-f5f0-4b5e-90fb-84694d6c32f4/Screenshot20240624at43731PM.png" />
+
+## Usage
+
+### Overview
+
+You can search and filter for data assets such as Cubes, Views, tables, charts, and dashboards.
+Semantic Catalog will show you an overview of each data asset, including information such as name,
+description, the measures and dimensions included (for a Cube or View), and usage information.
+
+### Lineage
+
+Semantic Catalog can be used to explore lineage between your connected data assets.
+For example, for a View, users can see what tables and Cubes flow into it as well as the downstream
+dashboards that query it in tools such as Tableau and Superset.
+
+<Screenshot src="https://ucarecdn.com/9adbeb24-203d-4608-9351-fac348a01f51/Screenshot20240624at44511PM.png" />
+
+[ref-tableau]: /product/configuration/visualization-tools/tableau
```

Comment:
Do these actually lead anywhere?

Reply:
The Tableau and Superset links lead to their pages under our Visualization section.

Or are you asking about the screenshots? In which case yes, but I just put them in my personal Uploadcare account... I assume there's a shared account I should add them to instead?

Reply:
Ah, I see it now
I just tried entering those links manually and got 404, but on the preview site they work 👍